Ontology/Event/Event.json

```diff
@@ -2,6 +2,16 @@
   "@id": "dtmi:digitaltwins:rec_3_3:core:Event;1",
   "@type": "Interface",
   "contents": [
+    {
+      "@type": "Relationship",
+      "description": {
+        "en": "The entity responsible for generating or producing the event."
+      },
+      "displayName": {
+        "en": "Produced By"
+      },
+      "name": "producedBy"
```

Comment:
In REC-OWL we actually have "Observation" as subclass of Event, which in turn has a property "observationGeneratedBy" pointing at a Sensor. This was filtered out from DTDL REC because it seemed to be out-of-scope to the ADT/DTDL/Willow use cases. If such a linkage is now of interest, I'd propose we remove that filter and include the upstream properties instead. OK?

Reply:
I think this has a little different intended use. "Produced By" is meant to be used by events which I wouldn't call "Observations" since they are being produced by an active system/person such as a "Card Swipe Event" or "Elevator Trip Event". The use of Event enables scenarios where a system is producing telemetry that is a little more complex than Capability which has a simple value (boolean or number). I decided on the term "produce" because this is what the Event-driven architectures use such as Kafka and Event Hub.